Makefile

```diff
@@ -33,5 +36,21 @@ endif
 	setfacl -m other::rwx ./tmp/nexus-volume
 	$(CACHITO_COMPOSE_ENGINE) up
 
-test:
-	PATH="$PWD/venv/bin:$PATH" tox
+run-down run-stop:
+	$(CACHITO_COMPOSE_ENGINE) down $(DOWN_OPTS)
+
+run-build: run-down
+	$(CACHITO_COMPOSE_ENGINE) build
+
+run-start-over: run-build run-start
```

Comment:
I wanted to provide a `run` command while forcing a rebuild if that is something that people wanted to always call, but I don't know what the best name for it would be. Feel free to suggest other things/tell me that this is not necessary.

Reply:
Also can't think of anything great, maybe `run-from-scratch`?

Reply:
`run-rebuild`?

Reply:
nevermind, that is too close to `run-build`.  I think the issue that I had is that I am overloading `run` as a generic term for running the development environment, but I could change targets to things like `dev-build`, `dev-run`, `dev-rebuild`, `dev-rebuild-run`. If I do this, then the `run-start-over` would instead be `dev-rebuild-run`. Thoughts?

Reply:
That sounds good to me. Maybe the `dev-` prefix is not even necessary, but I have not seen that many Makefiles so I'll defer to you on that

Reply:
Fair enough. I was just thinking to namespace them, but `build` _is_ namespaced... building the dev environment since Python doesn't need building... :D

Reply:
Hmm, now that you mention it, it *could* be confusing... Maybe having the prefix is better after all :D

Reply:
We can always namespace later if needed. There's benefit in keeping the target names short IMO.

Reply:
I will keep them "shorter" for now. No need to prematurely optimize.